rabbitmq-server-4.0.yaml

```diff
@@ -21,7 +21,7 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - elixir
+      - elixir-1.17
```

Comment:
If/when 1.18 or newer is supported, how will we know/catch the change?

Reply:
I think @jamonation this is not the right solution. At the current state it's just to highlight the point of failure for rabbitmq's JSON formatter